src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user (e.g. reading inputs, showing outputs).
+ */
+public class Ui {
+    private String lines = "____________________________________________________________";
+
+    /**
+     * Read user's inputs.
+     * @return User input as a string.
+     */
+    public String readCommand() {
+        Scanner scanner = new Scanner(System.in);
+        return scanner.nextLine();
+    }
```

Comment:
Why do you make a Scanner object every time you call readCommand(). Perhaps you could make 1 Scanner object per Ui object and call Scanner::nextLine() each time you call readCommand() instead?

Reply:
I didn't realise that :) Thank you for the input!